sdk/package.json

```diff
@@ -0,0 +1,124 @@
+{
+  "name": "@wormhole-foundation/sdk",
+  "version": "0.4.4-beta.5",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/wormhole-foundation/connect-sdk.git"
+  },
+  "bugs": {
+    "url": "https://github.com/wormhole-foundation/connect-sdk/issues"
+  },
+  "homepage": "https://github.com/wormhole-foundation/connect-sdk#readme",
+  "directories": {
+    "test": "__tests__"
+  },
+  "license": "Apache-2.0",
+  "main": "./dist/cjs/index.js",
+  "types": "./dist/cjs/index.d.ts",
+  "module": "./dist/esm/index.js",
+  "author": "",
+  "description": "The core package for the Connect SDK, used in conjunction with 1 or more of the chain packages",
+  "files": [
+    "dist"
+  ],
+  "keywords": [
+    "wormhole",
+    "sdk",
+    "typescript",
+    "connect"
+  ],
+  "engines": {
+    "node": ">=16"
+  },
+  "sideEffects": false,
+  "scripts": {
+    "build:cjs": "tsc -p ./tsconfig.cjs.json",
+    "build:esm": "tsc -p ./tsconfig.esm.json",
+    "build": "npm run build:cjs && npm run build:esm",
+    "rebuild": "npm run clean && npm run build:cjs && npm run build:esm",
+    "clean": "rm -rf ./dist && rm -f ./*.tsbuildinfo",
+    "lint": "npm run prettier && eslint --fix",
+    "prettier": "prettier --write ./src",
+    "coverage": "jest --config ./jest.config.ts --coverage",
+    "docs": "typedoc"
+  },
+  "exports": {
+    ".": {
+      "import": "./dist/esm/index.js",
+      "require": "./dist/cjs/index.js"
+    },
+    "./evm": {
+      "import": "./dist/esm/evm.js",
+      "require": "./dist/cjs/evm.js"
+    },
+    "./solana": {
+      "import": "./dist/esm/solana.js",
+      "require": "./dist/cjs/solana.js"
+    },
+    "./cosmwasm": {
+      "import": "./dist/esm/cosmwasm.js",
+      "require": "./dist/cjs/cosmwasm.js"
+    },
+    "./aptos": {
+      "import": "./dist/esm/aptos.js",
+      "require": "./dist/cjs/aptos.js"
+    },
+    "./sui": {
+      "import": "./dist/esm/sui.js",
+      "require": "./dist/cjs/sui.js"
+    },
+    "./algorand": {
+      "import": "./dist/esm/algorand.js",
+      "require": "./dist/cjs/algorand.js"
+    }
+  },
+  "typesVersions": {
+    "*": {
+      "evm": [
+        "dist/esm/evm.d.ts"
+      ],
+      "solana": [
+        "dist/esm/solana.d.ts"
+      ],
+      "cosmwasm": [
+        "dist/esm/cosmwasm.d.ts"
+      ],
+      "algorand": [
+        "dist/esm/algorand.d.ts"
+      ],
+      "aptos": [
+        "dist/esm/aptos.d.ts"
+      ],
+      "sui": [
+        "dist/esm/sui.d.ts"
+      ]
+    }
+  },
+  "dependencies": {
+    "@wormhole-foundation/sdk-base": "0.4.4-beta.5",
```

Comment:
did you use a script to generate all of this? or are we just gonna be replace-all-ing these bad boys?

Reply:
for any _new_ packages we'd have to add them manually, for versions it will get updated with the `syncVersions` script